lamindb/curators/core.py

```diff
@@ -851,6 +865,92 @@ def __init__(
         self._columns_field = self._var_fields
 
 
+class SomaExperimentCurator(SlotsCurator):
```

Comment:
This doesn't have `standardize`?

Reply:
This should come from `SlotsCurator`.

Reply:
Yes, but tiledbsoma requires certain special logic for that.